templates/iot/auth-s2i/050-Deployment-service-auth.yaml

```diff
@@ -0,0 +1,85 @@
+---
+apiVersion: apps/v1beta1
+kind: Deployment
+metadata:
+  name: iot-auth-service
+  labels:
+    app: iot-core
+    component: iot-auth-service
+    enmasse-component: iot
+  annotations:
+    image.openshift.io/triggers: >
+      [
+        {
+          "from":
+            {
+              "kind":"ImageStreamTag",
+              "name":"iot-auth-service:latest"
+            },
+            "fieldPath":"spec.template.spec.containers[?(@.name==\"auth-service\")].image"
+        }
+      ]
```

Comment:
What I was imagining was that you could have this annotation and it would be ignored if you did not have an imagestreamtag that matched. Is that not the case?